prime-router/metadata/hl7_mapping/common/observation-result.yml

```diff
@@ -103,7 +103,7 @@ elements:
         entered-in-error: W
 
   - name: observation-result-status-cancelled
-    value: ["'X'"]
+    value: [ "'X'" ]
```

Comment:
Curious if this was done by hand (saw this in a few spots) or by a formatter?  Mostly asking because we should get everyone using the same formatter

Reply:
I couldn't find the ticket that I think already existed for this, but I'll find that and create it if it doesn't exist. We should really have format templates for the major file types included in the repo and documented as part of the contributing guidelines, so I'll make sure we have something to cover that.